qrenderdoc/Windows/PixelHistoryView.cpp

```diff
@@ -487,9 +487,27 @@ class PixelHistoryItemModel : public QAbstractItemModel
 
     float rangesize = (m_Display.rangeMax - m_Display.rangeMin);
 
-    float r = val.col.floatValue[0];
-    float g = val.col.floatValue[1];
-    float b = val.col.floatValue[2];
+    float r = 0.0f;
+    float g = 0.0f;
+    float b = 0.0f;
+    if(m_IsFloat)
```

Comment:
I don't think so. The previous code was enabling the background colour pen for `m_IsDepth` or `m_IsFloat`. This change enables the background colour pen for all inputs i.e. `m_IsDepth || m_IsUint || m_IsSint || m_IsFloat` which is always true because if `m_IsFloat` is true when `m_IsUint` and `m_IsSint` are both false.